Windows 64bit -- cannot open input file 'kernel32.lib'

After following all the windows install steps, I have begun running into this gem of an error

```
PrepareForBuild:
  Creating directory "Release\obj\posix-ext\".
InitializeBuildStatus:
  Creating "Release\obj\posix-ext\posix-ext.unsuccessfulbuild" because "AlwaysCreate" was specified.
ClCompile:
  C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\bin\x86_amd64\CL.exe /c /I"C:\Users\IEguy\.node-gyp\0.10.32\src" /I"C:\Users\IEguy\.node-gyp\0.10.32\deps\uv\include" /I"C:\Users\IEguy\.node-gyp\0.10.32\deps\v8\include" /Zi /nologo /W3 /WX- /Ox /Ob2 /Oi /Ot /Oy /GL /D WIN32 /D _CRT_SECURE_NO_DEPRECATE /D _CRT_NONSTDC_NO_DEPRECATE /D BUILDING_V8_SHARED=1 /D BUILDING_UV_SHARED=1 /D BUILDING_NODE_EXTENSION /D _WINDLL /GF /Gm- /MT /GS /Gy /fp:precise /Zc:wchar_t /Zc:forScope /GR- /Fo"Release\obj\posix-ext\\" /Fd"Release\obj\posix-ext\vc100.pdb" /Gd /TP /wd4251 /errorReport:queue "..\src\posix-ext.cc" "..\src\process-win.cc" "..\src\fs-win.cc" "..\src\posix-win.cc" ..\src\autores.cc ..\src\winwrap.cc /MP
  posix-ext.cc
  process-win.cc
C:\Users\IEguy\.node-gyp\0.10.32\deps\v8\include\v8.h(179): warning C4506: no definition for inline function 'v8::Persistent<T> v8::Persistent<T>::New(v8::Handle<T>)' [C:\cygwin64\home\IEguy\repos\node-posix-ext\build\posix-ext.vcxproj]
          with
          [
              T=v8::Object
          ]
  fs-win.cc
C:\Users\IEguy\.node-gyp\0.10.32\deps\v8\include\v8.h(184): warning C4506: no definition for inline function 'v8::Persistent<T> v8::Persistent<T>::New(v8::Handle<T>)' [C:\cygwin64\home\IEguy\repos\node-posix-ext\build\posix-ext.vcxproj]
          with
          [
              T=v8::Object
          ]
  posix-win.cc
C:\Users\IEguy\.node-gyp\0.10.32\deps\v8\include\v8.h(179): warning C4506: no definition for inline function 'v8::Persistent<T> v8::Persistent<T>::New(v8::Handle<T>)' [C:\cygwin64\home\IEguy\repos\node-posix-ext\build\posix-ext.vcxproj]
          with
          [
              T=v8::Object
          ]
  autores.cc
C:\Users\IEguy\.node-gyp\0.10.32\deps\v8\include\v8.h(179): warning C4506: no definition for inline function 'v8::Persistent<T> v8::Persistent<T>::New(v8::Handle<T>)' [C:\cygwin64\home\IEguy\repos\node-posix-ext\build\posix-ext.vcxproj]
          with
          [
              T=v8::Object
          ]
  winwrap.cc
Link:
  C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\bin\x86_amd64\link.exe /ERRORREPORT:QUEUE /OUT:"C:\cygwin64\home\IEguy\repos\node-posix-ext\build\Release\posix-ext.node" /INCREMENTAL:NO /NOLOGO kernel32.lib user32.lib gdi32.lib winspool.lib comdlg32.lib advapi32.lib shell32.lib ole32.lib oleaut32.lib uuid.lib odbc32.lib DelayImp.lib "C:\Users\IEguy\.node-gyp\0.10.32\Release\node.lib" netapi32.lib /MANIFEST /ManifestFile:"Release\obj\posix-ext\posix-ext.node.intermediate.manifest" /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /DEBUG /PDB:"C:\cygwin64\home\IEguy\repos\node-posix-ext\build\Release\posix-ext.pdb" /OPT:REF /OPT:ICF /LTCG /TLBID:1 /DYNAMICBASE /NXCOMPAT /MACHINE:X64 /DLL "Release\obj\posix-ext\posix-ext.obj"
  "Release\obj\posix-ext\process-win.obj"
  "Release\obj\posix-ext\fs-win.obj"
  "Release\obj\posix-ext\posix-win.obj"
  "Release\obj\posix-ext\autores.obj"
  "Release\obj\posix-ext\winwrap.obj"
LINK : fatal error LNK1181: cannot open input file 'kernel32.lib' [C:\cygwin64\home\IEguy\repos\node-posix-ext\build\posix-ext.vcxproj]
Done Building Project "C:\cygwin64\home\IEguy\repos\node-posix-ext\build\posix-ext.vcxproj" (default targets) -- FAILED.
Done Building Project "C:\cygwin64\home\IEguy\repos\node-posix-ext\build\binding.sln" (default targets) -- FAILED.

Build FAILED.

"C:\cygwin64\home\IEguy\repos\node-posix-ext\build\binding.sln" (default target) (1) ->
"C:\cygwin64\home\IEguy\repos\node-posix-ext\build\posix-ext.vcxproj" (default target) (2) ->
(ClCompile target) ->
  C:\Users\IEguy\.node-gyp\0.10.32\deps\v8\include\v8.h(179): warning C4506: no definition for inline function 'v8::Persistent<T> v8::Persistent<T>::New(v8::Handle<T>)' [C:\cygwin64\home\IEguy\repos\node-posix-ext\build\posix-ext.vcxproj]
  C:\Users\IEguy\.node-gyp\0.10.32\deps\v8\include\v8.h(184): warning C4506: no definition for inline function 'v8::Persistent<T> v8::Persistent<T>::New(v8::Handle<T>)' [C:\cygwin64\home\IEguy\repos\node-posix-ext\build\posix-ext.vcxproj]
  C:\Users\IEguy\.node-gyp\0.10.32\deps\v8\include\v8.h(179): warning C4506: no definition for inline function 'v8::Persistent<T> v8::Persistent<T>::New(v8::Handle<T>)' [C:\cygwin64\home\IEguy\repos\node-posix-ext\build\posix-ext.vcxproj]
  C:\Users\IEguy\.node-gyp\0.10.32\deps\v8\include\v8.h(179): warning C4506: no definition for inline function 'v8::Persistent<T> v8::Persistent<T>::New(v8::Handle<T>)' [C:\cygwin64\home\IEguy\repos\node-posix-ext\build\posix-ext.vcxproj]


"C:\cygwin64\home\IEguy\repos\node-posix-ext\build\binding.sln" (default target) (1) ->
"C:\cygwin64\home\IEguy\repos\node-posix-ext\build\posix-ext.vcxproj" (default target) (2) ->
(Link target) ->
  LINK : fatal error LNK1181: cannot open input file 'kernel32.lib' [C:\cygwin64\home\IEguy\repos\node-posix-ext\build\posix-ext.vcxproj]

    4 Warning(s)
    1 Error(s)

Time Elapsed 00:00:03.16
```

Any insights appreciated.. the number of issues around this is pretty slim :|

thanks for everything you guys do :heart: 

--------------

#### Related reading 
#378
https://github.com/brianmcd/contextify/issues/110
https://github.com/prantlf/node-posix-ext/issues/1
http://stackoverflow.com/questions/9821700/visual-studio-2010-express-win-sdk-cannot-open-input-file-kernel32-lib

--- a/README.md
+++ b/README.md
@@ -40,12 +40,11 @@ You will also need to install:
     * [Python][windows-python] ([`v2.7.3`][windows-python-v2.7.3] recommended, `v3.x.x` is __*not*__ supported)
       * Make sure that you have a PYTHON environment variable, and it is set to drive:\path\to\python.exe not to a folder.
     * Windows XP/Vista/7:
-      * Microsoft Visual Studio C++ 2010 ([Express][msvc2010] version works well)
+      * Microsoft Visual Studio C++ 2013 ([Express][msvc2013] version works well)
         * If the install fails, try uninstalling any C++ 2010 x64&x86 Redistributable that you have installed first.
         * If you get errors that the 64-bit compilers are not installed you may also need the [compiler update for the Windows SDK 7.1]
     * Windows 7/8:
-      * Microsoft Visual Studio C++ 2012 for Windows Desktop ([Express][msvc2012] version works well)
-      * Microsoft Visual Studio C++ 2013 will also work.
+      * Microsoft Visual Studio C++ 2013 for Windows Desktop ([Express][msvc2013] version works well)
     * All Windows Versions
       * For 64-bit builds of node and native modules you will _**also**_ need the [Windows 7 64-bit SDK][win7sdk]
       * You may need to run one of the following commands if your build complains about WindowsSDKDir not being set, and you are sure you have already installed the SDK:
@@ -183,7 +182,6 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 [windows-python]: http://www.python.org/getit/windows
 [windows-python-v2.7.3]: http://www.python.org/download/releases/2.7.3#download
-[msvc2010]: http://go.microsoft.com/?linkid=9709949
-[msvc2012]: http://go.microsoft.com/?linkid=9816758
+[msvc2013]: http://www.visualstudio.com/en-us/downloads/download-visual-studio-vs
 [win7sdk]: http://www.microsoft.com/en-us/download/details.aspx?id=8279
 [compiler update for the Windows SDK 7.1]: http://www.microsoft.com/en-us/download/details.aspx?id=4422

